packages/application-config/src/process-config.ts

```diff
@@ -49,15 +49,27 @@ const processConfig = ({
   const additionalAppEnv = appConfig.additionalEnv ?? {};
   const revision = (additionalAppEnv.revision as string) ?? '';
 
-  const appEnvConfig = isProd ? appConfig.env.production : developmentConfig;
+  // Feature flags
+  const isOidcForDevelopmentEnabled =
+    processEnv.ENABLE_OIDC_FOR_DEVELOPMENT === 'true' ||
+    // @ts-expect-error
+    processEnv.ENABLE_OIDC_FOR_DEVELOPMENT === true;
```

Comment:
You might be able to just `JSON.parse(processEnv.ENABLE...)` to get the `string | boolean` into what you want: boolean.